applications/plugins/image/ImageFilter.h

```diff
@@ -642,7 +642,7 @@ class ImageFilter : public core::DataEngine
             {
                 unsigned int axis=0; if(p.size()) axis=(unsigned int)p[0];
                 if(axis==0) cimglist_for(img,l) img(l)=inimg(l).get_mirror ('x');
-                else if(axis==0) cimglist_for(img,l) img(l)=inimg(l).get_mirror ('y');
+                else if(axis==1) cimglist_for(img,l) img(l)=inimg(l).get_mirror ('y');
```

Comment:
maybe would be nice to put more readable if / else if scope with  { }  and return line.

Reply:
I was afraid someone would say that 😱

Reply:
it was "couru d'avance"

Reply:
Ready it is